execution-environment.yml

```diff
@@ -12,15 +12,11 @@ dependencies:
   python: requirements.txt
   galaxy: requirements.yml
 
-additional_build_files:
```

Comment:
Any thoughts on keeping an ansible.cfg in here? See https://github.com/opdev/opcap-ansible/issues/110.

Reply:
we can scratch the idea of packing in the `ansible.cfg` if custom forks value isn't a pre-req for proper playbook execution.

>>  I'm not even sure that an ansible.cfg in the EE would even work

Based off my experience experimenting with `ansible-navigator config`, specifying an eei will cause navigator to use the `/etc/ansible/ansible.cfg` residing within the eei. If it's not there, it will default to `ansible.cfg` on the local filesystem outside container.

Reply:
I'd say scratch it for now. The custom forks value should be tested for validity again.